cmd/gapit/commands.go

```diff
@@ -104,7 +104,7 @@ func (verb *commandsVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 
 	return traverseCommandTree(ctx, client, tree.Root, func(n *service.CommandTreeNode, prefix string) error {
 		if verb.OnlyExecutedDraws {
-			if n.Group != "" || n.NumChildren > 0 {
+			if n.Group != "" || n.NumChildren > 0 || len(n.Commands.First().Indices) == 1 {
```

Comment:
Could you add a comment to clarify why this additional check is necessary? (maybe explaining that some queueSubmit may pass the two other checks yet not be relevant for OnlyExecutedDraws)